### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by ibennetch**
* Grammar fixes
* Document additional ENV variables
  * Related to/fixes the comment at https://github.com/phpmyadmin/docker/issues/151#issuecomment-630246744

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,10 +14,11 @@ where you can enjoy your happy MySQL administration.
 
 ## Credentials
 
-phpMyAdmin does use MySQL server credential, please check the corresponding
-server image for information how it is setup.
+phpMyAdmin connects using your MySQL server credentials, please check the corresponding
+database server image for information on what the default username and password are
+or to modify these during installation.
```
Comment (williamdes) [inline README.md:19]: I had to read it multiple times to understand the `are or` could you split this in two phrases please ?

File: README.md [hunk 2]
```diff
@@ -130,6 +131,13 @@ Set the variable ``PMA_ABSOLUTE_URI`` to the fully-qualified path (``https://pma
 * ``UPLOAD_LIMIT`` - if set, will override the default value for apache and php-fpm (default value is 2048 kb)
 * ``PMA_CONFIG_BASE64`` - if set, will override the default config.inc.php with the base64 decoded contents of the variable
 * ``PMA_USER_CONFIG_BASE64`` - if set, will override the default config.user.inc.php with the base64 decoded contents of the variable
+* ``PMA_ABSOLUTE_URI`` - the full URL to phpMyAdmin. Sometimes needed when used in a reverse-proxy configuration. Don't set this unless needed
+* ``PMA_CONTROLHOST`` - when set, this points to an alternate database host used for storing the phpMyAdmin Configuration Storage database
```
Comment (williamdes) [inline README.md:135]:
```suggestion
* ``PMA_CONTROLHOST`` - when set, this points to an alternate database host used for storing the [phpMyAdmin Configuration Storage database](https://docs.phpmyadmin.net/en/latest/setup.html#phpmyadmin-configuration-storage)
```

Maybe a link to our docs could be a nice to have ?

Comment (ibennetch) [reply README.md:135]: Excellent suggestion. I forgot this was Markdown for a moment and that linking is easy. I'll do this.

File: README.md (hunk 1 above)
Comment (ibennetch) [reply README.md:19]: Good suggestion, this is a bit too awkward.

**Comment by williamdes** [comment]
:ping_pong: @ibennetch do you want to finish this one before the next 5.0.x release ?

**Pull request merged**